force-app/main/default/triggers/JobApplicationTrigger.trigger

```diff
@@ -0,0 +1,5 @@
+trigger JobApplicationTrigger on Job_Application__c (before insert, after insert, before update, after update, before delete, after delete, after undelete) {
+    JobApplicationHandler handler = new JobApplicationHandler();
+
```

Comment:
fix spacing, I would not put a space between these two lines of code and I would put a space before and after for clean and easy to read code.